server/routes/tests/post.todo.test.js

```diff
@@ -0,0 +1,91 @@
+const request = require("supertest");
+const app = require("../../app");
+const database = require("./../../firebase").database;
+
+let NZGmt = 13;
+let todaysDate = new Date();
+todaysDate.setHours(todaysDate.getHours() + NZGmt);
+todaysDate = todaysDate.toISOString().slice(0, 10);
+
+const expectedToDoListData = [
```

Comment:
It would be better to import  mockToDoListData from mockDatabase.
Then you can make expectedToDoListData = mockToDoListData  + the todolistItemInput 
By doing this you don't have to worry about future changes to the mockToDoListData breaking your tests.

Otherwise everything looks great! :)